libfwupdplugin/fu-fit-firmware.c

```diff
@@ -244,7 +244,7 @@ fu_fit_firmware_verify_image(FuFirmware *firmware,
 	if (g_getenv("FU_FDT_FIRMWARE_VERBOSE") != NULL)
 		fu_dump_bytes(G_LOG_DOMAIN, "data", blob);
 
-	/* verify any hashes we recoginise */
+	/* verify any hashes we recognize */
```

Comment:
Google picked the `-ize` ending (and I consciously chose not to intervene). If this project has a strong `-ise` preference, I can certainly fix that.

Reply:
Although I'm en_GB, I think we use en_US everywhere